_layouts/default.html

```diff
@@ -6,7 +6,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>{{ page.title }}</title>
 	<link rel="stylesheet" href="{{ "/assets/style.css" | relative_url }}">
-	<link rel="stylesheet" href="https://campuspulse.app/css/Menu.css">
+	<link rel="stylesheet" href="../assets/Menu.css">
```

Comment:
Can you update this link to use the same style as the stylesheet in the line above? this will allow jekyll to generate correct links, rather than hardcoding it